docs/packaging-a-RNComponent-ios.md

```diff
@@ -0,0 +1,77 @@
+---
+id: packaging-a-RNComponent-ios
+title: Packaging a react-native component
+layout: docs
+category: Guides (iOS)
+permalink: docs/packaging-a-RNComponent-ios.html
+next: native-modules-android
+---
+
+After you've written some code in an existing project and it works great
+and you want to pull it out into its own package, put it on npm, then
+install it back into your project. But wait, your component is partially
+implemented in Objective-C! How can you share this on `npm`? 
+Please read the following steps
+### Creating the package
+
+1. Create a new project, use this type of naming, RN prefix.
+
+![new-project](http://brentvatne.ca/images/packaging/1-new-project.png)
+![new-project-2](http://brentvatne.ca/images/packaging/2-project-name.png)
+**RN is a prefix that I have been using as RCT should only be used for
+the react-native standard library, feel free to adopt it or use your
+own**
+
+2. `cd` to the project directory and run `npm init`, fill out your info!
+
+![npm-init](http://brentvatne.ca/images/packaging/3-npm-init.png)
+
+3. `npm install react-native --save-dev`, then open up `package.json`
+   and also add the same version to `peerDependencies`.
```

Comment:
Isn't the `most-up-to-date` approach to just specify `peerDependencies: { "react-native": "*" }`?